examples/demo/tests/cmd/scheduler.trycmd

```diff
@@ -0,0 +1,19 @@
+```console 
+$ LOCO_ENV=test blo-cli scheduler --list
+#      job_name        cron               tags               kind
+1      Run command     */1 * * * * *      base, infra        Shell { command: "echo loco >> ./scheduler.txt" }
+2      Run command     */5 * * * * *      -                  Task { name: "foo", vars: None }
+3      list if users   */7 * * * * *      base, users        Task { name: "user_report", vars: None }
+
+
+```
+
+```console 
+$ LOCO_ENV=test blo-cli scheduler --config ./config/scheduler.yaml --list
+#      job_name        cron               tags               kind
+1      add text        */1 * * * * *      base, infra        Shell { command: "echo loco >> ./scheduler.txt" }
```

Comment:
there are some libraries that do "english to cron" and "cron to english"
might be good to add a "cron to english" description here, based on the cron expression that's saved in the configuration